src/herder/test/HerderTests.cpp

```diff
@@ -1406,7 +1406,8 @@ TEST_CASE("quick restart", "[herder][quickRestart]")
 
     // SMALL_GAP happens to be the maximum number of ledgers
     // that are kept in memory
-    auto static const SMALL_GAP = Herder::MAX_SLOTS_TO_REMEMBER + 1;
+    auto static const SMALL_GAP =
+        simulation->getApp()->getConfig().MAX_SLOTS_TO_REMEMBER + 1;
```

Comment:
Sorry, I missed this one: you should not use the simulation's idle app, and instead use one of the nodes, in this case, `simulation->getNode(listenerKey.getPublicKey())` (that way you can remove the accessor to the idle app from simulation)